style.css

```diff
@@ -0,0 +1,88 @@
+.clearfix::before,
+.clearfix::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+
+body {
+  margin: 0 auto;
+  font-size: 18px;
+  max-width: 800px;
+  min-width: 670px;
+  color: rgb(126, 126, 126);
+  background-color: rgb(207, 207, 207);
+  padding: 10%;
+}
+
+.border {
+  background-color: rgb(255, 252, 248);
+  padding: 10%;
+  border: 1px transparent;
+  border-radius: 100px;
+}
+
+h2,
+h3,
+b {
+  color: rgb(90, 90, 90);
+  margin-top: 5%;
+  padding-top: 0;
```

Comment:
The padding here reiterates the  [default browser styles](https://browserdefaultstyles.com/) for these and, therefore, can be removed:
```suggestion
```